src/main/java/xaeroplus/mixin/client/MixinGuiMinimapOverlaysSettings.java

```diff
@@ -0,0 +1,31 @@
+package xaeroplus.mixin.client;
+
+import net.minecraft.client.gui.GuiScreen;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import xaero.common.IXaeroMinimap;
+import xaero.common.gui.*;
+import xaeroplus.settings.XaeroPlusSettingRegistry;
+
+@Mixin(value = GuiMinimapOverlaysSettings.class, remap = false)
```

Comment:
Non-blocking: I think we should just move these to the main minimap settings page so we can add other settings that don't fit in this category. Or even better (but more work) - make the sub-setting page configurable on the setting definition.

One setting I'm working on adding soon:tm: is a configurable minimap zoom much lower than the default